Add execution time limit to subroutines (#219)
timewave-computer/valence-protocol: docs/src/authorizations_processors/processor.md

### Execution

When a processor is `Ticked`, the first `Message Batch` will be taken from the queue (`High` if there are batches there or `Med` if there aren’t).

> After taking it, the first check the processor will do is check if the batch is expired. If that's the case, we will discard the batch and return an `Expired(executed_functions)` `ExecutionResult` to the Authorization contract. There might be a case that the batch is `NonAtomic` and it's already partially executed, therefore we return the number of functions that were executed before the expiration.

Reviewer: ```suggestion
After taking it, the first check the processor will do is check if the batch is expired. If that's the case, the processor will discard the batch and return an `Expired(executed_functions)` `ExecutionResult` to the Authorization contract. There might be a case that the batch is `NonAtomic` and it's already partially executed, therefore we return the number of functions that were executed before the expiration.
```

> After taking it, the first check the processor will do is check if the batch is expired. If that's the case, we will discard the batch and return an `Expired(executed_functions)` `ExecutionResult` to the Authorization contract. There might be a case that the batch is `NonAtomic` and it's already partially executed, therefore we return the number of functions that were executed before the expiration.

Reviewer 2: ```suggestion
After taking the `Message Batch`, the processor will first check if the batch is expired. If that's the case, the processor will discard the batch and return an `Expired(executed_functions)` `ExecutionResult` to the Authorization contract. There might be a case that the batch is `NonAtomic` and it's already partially executed, therefore the processor also returns the number of functions that were executed before the expiration.
```

> If it's still valid, the processor will execute it in different ways depending if the batch is `Atomic` or `NonAtomic`.

Reviewer 2: ```suggestion
If the batch has not expired, the processor will execute the batch according to whether it is `Atomic` or `NonAtomic`.
```
